src/main/scala/com/sky/kafka/configurator/Main.scala

```diff
@@ -32,55 +30,32 @@ object Main extends LazyLogging {
   def main(args: Array[String]): Unit = {
     logger.info(s"Running ${BuildInfo.name} ${BuildInfo.version} with args: ${args.mkString(", ")}")
     run(args) match {
-      case Success(_) =>
-        System.exit(0)
-      case Failure(_) =>
+      case Success((errors, infoLogs)) =>
+        errors.foreach(e => logger.warn(e.getMessage))
+        infoLogs.foreach(msg => logger.info(msg))
```

Comment:
I much prefer logging these here

Reply:
I think it would be useful to clearly list the successfully configured topics and then list the unsuccessfully configured topics (maybe can just be added into success log strings but we should def look at using the topic name on the error)

Reply:
the error type we always return from `KafkaConfiguratorApp` is `ConfiguratorFailure` which contains the topic name in the message already (check `error` package object). All the info log messages we generate already have the topic names too e.g.

```
Running kafka-configurator 0.7.0-SNAPSHOT with args: -f, /topic-config.yml, --zookeeper, kafka:2181
Failed to configure topic1. Cause: Changing replication factor is unsupported
Failed to configure topic2. Cause: replication factor: 2 larger than available brokers: 1
Replication factor unchanged for topic3.
No change in number of partitions for topic3
No change in config for topic3
```

Reply:
Ah I didn't see the topic name was used in the constructor of `Exception` for the message. Possibly it would be nice to have the topic names aligned in some way in the output and maybe some explicit grouping of failures and successes but I am being very picky we can def do another time if needed 😛